src/reducers/picklist/__tests__/PicklistReducer-test.js

```diff
@@ -482,4 +483,71 @@ describe('PicklistReducer', () => {
     }
     assert.deepStrictEqual(newState, expectedState, 'Should not mark any item')
   })
+
+  it('Refresh node updates path', () => {
+    const newState = picklistReducer(testState, {
+      type: LOAD_ONE_NODE_SUCCESS,
+      result:{
+        path: ',1,',
+        pathNames: [{ id: 1}] },
+      id: 1
+    })
+    const expectedState =  {
+      NODE: [
+        {
+          marked: false,
+          path: [],
+          value: {
+            ...node1
+          }
+        },
+        {
+          marked: false,
+          path: [],
+          value: node2
+        },
+        {
+          marked: false,
+          path: [],
+          value: node3
+        }
+      ],
+      OBJECT: DEFAULT_OBJECTS
+    }
+    assert.deepStrictEqual(JSON.stringify(expectedState), JSON.stringify(newState), 'Should update path for node 1')
+  })
+
+it('Refresh node updates path works', () => {
+  const newState = picklistReducer(testState, {
+    type: LOAD_ONE_NODE_SUCCESS,
+    result:{
+      path: ',1,2,3,',
+      pathNames: [{ id: 1},{ id: 2},{ id: 3}] },
+    id: 1
+  })
+  const expectedState =  {
+    NODE: [
+      {
+        marked: false,
+        path: [null],
```

Comment:
this seems to be a bug for me. why is path expected to be array with one null element when you modify the state with a result of path with path and pathnames?